README.md

```diff
@@ -119,6 +138,7 @@ Add the custom component to the page:
 
 `<labeled-input></labeled-input>`

```

Comment:
This is just a random thought, feel free to ignore it. We have in the documentation examples using the labeled-input component and it seemed to me like we actually have this component and I had to double check :D 

Of course that could just be me, but wanted to point it out if there is a chance people will get confused and decide to look through the code for a labeled-input component.

Reply:
It makes sense. It can be confusing. Replaced the <labeled-input> with the <checkbox>. For the sake of simplicity removed part of the code, but added a link to the whole example so that people can check it.